src/main/java/ArchipelagoMW/APClient.java

```diff
@@ -144,6 +149,19 @@ public void onConnectResult(ConnectionResultEvent connectionResultEvent) {
 
             LocationTracker.scoutFirstLocations();
 
+            Set<Long> checkedLocations = getLocationManager().getCheckedLocations();
+            NeowPatch.act2portalAvailable = false;
```

Comment:
This can be simplified to
`NeowPatch.act2portalAvailable = checkedLocations.contains(22001L)` // check if act1 boss relic location has been sent.

Same goes for the other act portal check.